course_info/icommons.py

```diff
@@ -150,6 +191,24 @@ def get_school_info(self, school_id):
                 logger.error(e.message)
         return school_info
 
+    def get_course_info_instructor_list(self, course_instance_id=None):
+        """
+            retrieves course instructors information from the iCommons API
+            or returns None if there are no instructors
+        """
+        cache_key = CACHE_KEY_COURSE_PEOPLE_BY_COURSE_INSTANCE_ID.format(course_instance_id)
+        course_staff_info = cache.get(cache_key)
+        if course_staff_info is None:
+            course_staff_info = {}
```

Comment:
If `_course_info_instructor_list()` returns a list, should this be initialized as an empty list?